tests/openvino/utils_tests.py

```diff
@@ -172,7 +172,7 @@
     "xlm_roberta": "hf-internal-testing/tiny-xlm-roberta",
     "xglm": "hf-internal-testing/tiny-random-XGLMForCausalLM",
     "xverse": "katuni4ka/tiny-random-xverse",
-    "glm4": "katuni4ka/tiny-random-glm4",
+    "glm4": "snake7gun/tiny-random-glm4",
```

Comment:
@eaidova fyi

Reply:
I would like to have both, not just replacing one by another, please rename key for my model in tests

Reply:
done, pls help to check if it make sense